_gabarits-jsonld/README.md

```diff
@@ -66,6 +72,15 @@ Saisissez les informations identifiant et décrivant la compagnie, le groupe ou
 ### [_location_](https://schema.org/location)
 Saisissez les informations identifiant et décrivant l'endroit où est présenté l'événement. L'entité décrite sous la propriété _location_ peut être de @type [Place](https://schema.org/Place) (un lieu physique) ou de @type [VirtualLocation](https://schema.org/VirtualLocation) (un lieu virtuel).
 
+### [_sameAs_](https://schema.org/sameAs)
+Saisissez les URIs d'_identifiants passerelles_ permettant d'identifier sans ambiguité les _entitées imbriquées_ dans l'entité de type _Event_. 
+
+Par « entités imbriquées », on entend des entités liées à l'entité _Event_ par le biais des propriétés telles _organizer_, _performer_ et _location_. Ces entités sont des [choses](https://schema.org/Thing) à part entières. Elles peuvent avoir un _@type_ différent de l'entité principale. Elles peuvent être décrites avec des pairs propriété-valeur au même titre que l'entité principale. En imbriquant ces entités à l'intérieur du code d'une entité principale à l'aide d'accolades `{ }` : on signifie que l'entité de premier niveau est l'entité principale; on décrit sémantiquement la relation entre l'entité de premier niveau et ses entités de second niveau; et on évite toute confusion à savoir si une pair propriété-valeur (par exemple, un nom ou une description) se rapporte à l'entité principale ou à une entité secondaire. Les entités imbriquées sont généralement représentées (dans le code et dans les visualisateurs JSON-LD) avec une indentation, cela afin de faciliter la lecture du code par les humains.
+
+Par « identifiants passerelles », on entend des identifiants pérennes (c.-à-d. des URIs) globalement uniques qui sont susceptibles d'être utilisés par un grand nombre de systèmes d'informations et qui peuvent ainsi faciliter la réconciliation des entités entre différents jeux de données. Les identifiants ISNI, Wikidata et Artsdata sont particulièrement pertinents dans le domaine du spectacle.
+
+Saississez toujours l'URI complète de l'identifiant (plutôt que de saisir uniquement l'identifiant lui-même). Par exemple, pour l'identifiant Wikidata Q596774 correspond l'URI http://www.wikidata.org/entity/Q596774 (prenez note que le format de l'URI diffère de l'URL de la page). Pour plus d'informations, consultez ces [recommandations à propos des identifiants pérennes dans les arts de la scène](https://docs.google.com/spreadsheets/d/1j2Be-KBZm4LioW3DH2NU7GR3m77boTeQcIHZe8OMK3U/edit?usp=sharing). 
```

Comment:
This is very helpful to mention. Thanks. Maybe an example of ISNI would be good as well so people do not put only the ID.

Reply:
I thought about providing an ISNI example, but I felt the paragraph would be a little too long. In addition, it seemed more appropriate to provide detailed instructions (on URI formats and how to retrieve them) in the spreadsheet rather than in the structured data template. That way, we will only have a single authoritative resource about IDs rather than multiple bits of information here and there. (FYI - In my mind, the spreadsheet is meant to grow into a properly formatted resource that will exist in Github).

Reply:
Please note that I made another edit to this paragraph in my latest commit.